### PR TITLE
[RFC] vim-patch: Mark patches as NA

### PR DESCRIPTION
**Pull request by glts**
Some obvious NA patches (found by scanning the patch queue from the bottom up until around patch 7.4.650).
-   https://github.com/vim/vim/commit/v7-4-592
-   https://github.com/vim/vim/commit/v7-4-596
-   https://github.com/vim/vim/commit/v7-4-599
-   https://github.com/vim/vim/commit/v7-4-610
-   https://github.com/vim/vim/commit/v7-4-618
-   https://github.com/vim/vim/commit/v7-4-619
-   https://github.com/vim/vim/commit/v7-4-621
-   https://github.com/vim/vim/commit/v7-4-622
-   https://github.com/vim/vim/commit/v7-4-625
-   https://github.com/vim/vim/commit/v7-4-626
-   https://github.com/vim/vim/commit/v7-4-632
-   https://github.com/vim/vim/commit/v7-4-638
-   https://github.com/vim/vim/commit/v7-4-647
-   https://github.com/vim/vim/commit/v7-4-648
-   https://github.com/vim/vim/commit/v7-4-650

See the commit messages for details.

### CODE REVIEW

**Comment by Pyrohh** [comment]
I love the commit message format; it only took me a few minutes to review because of it.

Besides the nit on the second commit's title, LGTM.

File: src/nvim/version.c [hunk 1]
```diff
@@ -184,14 +184,14 @@ static int included_patches[] = {
   //602,
   601,
   //600,
-  //599,
+  //599 NA
```
Comment (justinmk) [inline src/nvim/version.c]: This should be uncommented, not NA.

**Comment by justinmk** [comment]
Nice, LGTM after comment

**Comment by glts** [comment]
Thanks. Changed as suggested, apart from uncommenting `599` only the commit messages were changed.

**Comment by Pyrohh** [comment]
Thanks @glts. You seem really motivated to get nvim up to date with patches :-)

**Comment by glts** [comment]
thanks, I suppose you could say that :smile:

**Pull request merged**